packages/bsky/src/config.ts

```diff
@@ -144,6 +152,22 @@ export class ServerConfig {
     const maxThreadDepth = process.env.BSKY_MAX_THREAD_DEPTH
       ? parseInt(process.env.BSKY_MAX_THREAD_DEPTH || '', 10)
       : undefined
+
+    const disableSsrfProtection =
+      process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
```

Comment:
Could we just configure this option in those environments rather than infer it?  Or base it on the value of `debugMode`?

Reply:
Changed the definition of `debugMode` so that is is "opt-in" instead of defaulting to true. Added the ability to override the value by setting the `BSKY_DISABLE_SSRF_PROTECTION` env var.